src/server.ts

```diff
@@ -25,26 +25,34 @@ const authenticationBasePath = container.resolve<string>(
 const APIBasePath = container.resolve<string>("APIBasePath");
 const PagoPABasePath = container.resolve<string>("PagoPABasePath");
 
-const app = newApp(
-  env,
-  allowNotifyIPSourceRange,
-  allowPagoPAIPSourceRange,
-  authenticationBasePath,
-  APIBasePath,
-  PagoPABasePath
-);
-
-// In test and production environments the HTTPS is terminated by the Kubernetes Ingress controller. In dev we don't use
-// Kubernetes so the proxy has to run on HTTPS to behave correctly.
-if (env === NodeEnvironmentEnum.DEVELOPMENT) {
-  const samlKey = container.resolve<string>(SAML_KEY);
-  const samlCert = container.resolve<string>(SAML_CERT);
-  const options = { key: samlKey, cert: samlCert };
-  https.createServer(options, app).listen(443, () => {
-    log.info("Listening on port 443");
-  });
-} else {
-  http.createServer(app).listen(port, () => {
-    log.info("Listening on port %d", port);
-  });
+function startApp(): void {
```

Comment:
what's the purpose of `startApp()`? it just calls `newApp`, can we get rid of it and just call `newApp`?

Reply:
Sorry, I wrote this function for a wrong reason and I forgot to remove that. Fixed in a9d3aeb